.travis.yml

```diff
@@ -22,20 +21,55 @@ branches:
   only:
     - master
 
+env:
+  global:
+    - PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
+    - LOCAL_DIR: build
+    - LOCAL_SCRIPT_DEBUG: false
+    - COMPOSER_INSTALL: false
+    - WORDPRESS: true
```

Comment:
It wasn't immediately clear to me what `WORDPRESS` does, could we name it `INSTALL_WORDPRESS` or something? (Could rename `COMPOSER_INSTALL` to `INSTALL_COMPOSER`, too, so that there's parity.)

Reply:
👍🏻 5c43925a5